src/app.js

```diff
@@ -49,3 +59,27 @@ ipc.on('application:quitting', () => {
     machine.stop();
   }
 });
+
+ipc.on('application:open-url', opts => {
+  var parser = document.createElement('a');
+  parser.href = opts.url;
+
+  if (parser.protocol !== 'docker:') {
+    return;
+  }
+
+  var pathname = parser.pathname.replace('//', '');
```

Comment:
It would be great to add comments with an example of a valid docker:// URL and what happens as we parse it.